src/background/modules/AutocorrectHandler.js

```diff
@@ -26,6 +26,76 @@ let antipatterns = [];
 // Adapted from: https://github.com/mozilla/webextension-polyfill/blob/master/src/browser-polyfill.js
 const IS_CHROME = Object.getPrototypeOf(browser) !== Object.prototype;
 
+/**
+ * Traverse Trie tree of objects to create RegEx.
+ *
+ * @param {Object.<string, Object|boolean>} obj
+ * @returns {string}
+ */
+function pattern(obj) {
+    const alt = [];
+    const cc = [];
```

Comment:
What's cc? Again please use a readable variable name.

Reply:
`alt` is also kinda unclear.

Reply:
cc = [Character Class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes)
alt = Alternation

These abbreviations were not my idea (see below), but I am happy to change them if you have a better suggestion.

Reply:
Well just do not abbreviate them…

```suggestion
    const characterClass = [];
```

same with that alternation thing?

Then one can at least look that term up. :upside_down_face: